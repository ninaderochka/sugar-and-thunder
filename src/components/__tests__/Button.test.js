import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button';

test('Button component renders correctly', () => {
  const mockCallBack = jest.fn();
  const component = renderer.create(
    <Button 
    value="Test" 
    onClick={mockCallBack} 
    align="text-center"
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});