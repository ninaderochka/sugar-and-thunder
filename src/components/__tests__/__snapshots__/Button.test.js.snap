// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly 1`] = `
<button
  className="text-center font-poppins font-normal text-black text-2xl uppercase w-fit px-6 py-2.5 bg-button-blue leading-tight rounded shadow-md hover:bg-button-blue/90 hover:shadow-lg focus:bg-button-blue/90 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-button-blue/90 active:shadow-lg transition duration-150 ease-in-out"
  onClick={[Function]}
  type="button"
>
  Test
</button>
`;
